networkx/classes/digraph.py

```diff
@@ -732,6 +732,13 @@ def has_successor(self, u, v):
         """
         return (u in self.succ and v in self.succ[u])
 
+    def has_successors(self, u, v):
+        """Return True if node u has any successors, otherwise False."""
+        try:
+            return bool(self.succ[u])
```

Comment:
Though it is correct, I find casting a dictionary to a Boolean a little mysterious. It's a very C-like thing to do. `return len(self.succ[u]) == 0` is clearer. I don't feel too too strongly about that though.